integration/util/request/client.go

```diff
@@ -13,3 +19,115 @@ func NewAPIClient(t *testing.T) client.APIClient {
 	require.NoError(t, err)
 	return clt
 }
+
+// Create creates a container with the supplied parameters
+func Create(client client.APIClient, v []string, memoryMB int, labels map[string]string, img string, cmd []string) (string, error) {
```

Comment:
I don't think the helpers added to this package should be exported from a package. Please move these to the `integration/plugin` package. 

We already have an abstraction for making API calls (the client), any helpers are better off being suite specific instead of trying to maintain another abstraction.